admin/partials/requests.php

```diff
@@ -20,9 +20,9 @@
 	<div class="nav-tab-wrapper">
 		<?php foreach ( $tabs as $key => $value ) : ?>
 			<a href="<?php echo esc_url( '#' . $key ); ?>" class="nav-tab">
-				<?php echo esc_html( $value['name'] ); ?>
+				<?php echo esc_html_e( $value['name'] ); ?>
```

Comment:
Do you need to `echo` an `esc_html_e`?

Pretty sure the last `e` is for `echo`: https://developer.wordpress.org/reference/functions/esc_html_e/#source

Reply:
Not going to comment on the ones in the rest of the file. A search and replace will do the job here

Reply:
Updated to remove all the echo's where esc_html_e() is used